modules/sagemaker/sagemaker-custom-kernel/modulestack.yaml

```diff
@@ -0,0 +1,61 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This stack deploys a Module specific IAM permissions
+
+Parameters:
+  ProjectName:
+    Type: String
+    Description: The name of the project
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+  StudioExecutionRoleArn:
+    Type: String
+    Description: The name of the IAM Role
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Effect: Allow
+            Action:
+              - "ecr:Describe*"
+              - "ecr:Get*"
+              - "ecr:List*"
+              - "sagemaker:CreateImage"
+              - "sagemaker:CreateImageVersion"
+              - "sagemaker:DescribeImage"
+              - "sagemaker:DescribeImageVersion"
+              - "sagemaker:DescribeAppImageConfig"
+              - "sagemaker:CreateAppImageConfig"
+              - "sagemaker:UpdateAppImageConfig"
+              - "sagemaker:UpdateDomain"
+              - "sagemaker:DescribeDomain"
+              - "sagemaker:AddTags"
+            Resource: "*"
```

Comment:
is this correct?  All resources?  Or just this module role?

Reply:
Good call, will limit resources + not all actions here are required since this is using L1 CDK constructs for app config instead of a custom script.

Reply:
Done